connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/User.java

```diff
@@ -0,0 +1,162 @@
+package com.redhat.cloud.notifications.connector.email.model;
+
+import io.vertx.core.json.JsonObject;
+
+import java.util.Objects;
+
+/**
+ * Represents the internal user model used to effectively unify and process the
+ * users from the user providers.
+ */
+public final class User {
+
+    private String id;
+    private String username;
+    private String email;
+    private String firstName;
+    private String lastName;
+    private boolean isActive;
+    private boolean isAdmin;
```

Comment:
`id`, `email`, `firstName` and `lastName` are never used and there is no reason to believe this will change in the near future. We could make the code simpler and remove these fields.